src/main/java/za/co/neslotech/spring/trainspringbootinsight/model/Transaction.java

```diff
@@ -24,6 +25,9 @@ public class Transaction {
     @Column(length = 19, precision = 4)
     private Double amount;
 
+    @ManyToOne
+    private Transaction transaction;
```

Comment:
Is this correct to say many transactions can have a single transaction? I don't quite understand this relationship?